openesdh-repo/src/main/amp/config/alfresco/module/openesdh-repo/models/doc-model.xml

```diff
@@ -290,6 +282,12 @@
                 </property>
             </properties>
         </type>
+        
+        <!-- other types -->
+        <type name="doc:dtype">
+            <title>Document type</title>
+            <parent>cm:content</parent>
```

Comment:
Perhaps the parent type for classification values could be different, since classification values don't have any content to store. Thus the instances would be more lightweight.